plugins/woocommerce/includes/admin/class-wc-admin-menus.php

```diff
@@ -246,32 +248,75 @@ public function menu_highlight() {
 				}
 				break;
 		}
+
+		// Highlight "Add new order" submenu when on the new order page.
+		if ( isset( $_GET['page'], $_GET['action'] ) && 0 === strpos( sanitize_text_field( wp_unslash( $_GET['page'] ) ), 'wc-orders' ) && 'new' === sanitize_text_field( wp_unslash( $_GET['action'] ) ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			$submenu_file = add_query_arg( 'action', 'new', admin_url( 'admin.php?page=' . sanitize_text_field( wp_unslash( $_GET['page'] ) ) ) ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited,WordPress.Security.NonceVerification.Recommended
+		}
```

Comment:
Repetitive `$_GET['page']` sanitization - looks like this can be simplified.

Reply:
Why is `$submenu_file` getting set to a random value for `$_GET['page']` - wouldn't this be a known value (or narrow set of known values)?